### ISSUE TITLE
about training problem.

### ISSUE THREAD

**Issue by DavidKong96** [hytseng0509/CrossDomainFewShot#22]
Thanks for your sharing. 
There is a question about should i train the pre-trained encoder from 0 ,or i can transfer the weights of  resnet10/18/34 to train the model. Will that have any impact? Such as decreased accuracy, overfitting？
And is ResNet34  performs worse than resnet10 or 18?
I just got in touch with this cross-domain not long，hope you can reply.It will help me a lot. Thank you so much!

**Comment by hytseng0509** [comment]
You should train the pre-trained encoder using only the training categories in our setting. As the pre-trained model by torchvision may use the categories that serves as our testing categories to train the model, you should train the encoder from scratch.
According to Table A5 in this [paper](https://arxiv.org/pdf/1904.04232.pdf), the model size does not have a significant impact on the few-shot classification performance.
Hope that answers your questions.

**Comment by DavidKong96** [comment]
ok，thanks. i will try it .It's very helpful
